### ISSUE TITLE
Update & document 'glossary.tex'

### ISSUE THREAD

**Issue by ghost** [schnes4/dhbw-heidenheim-latex-template#18]
Remove the currently content in _glossary.tex_ and document in the wiki how to use the file.

**Comment by nobodypb** [comment]
We should also think about using template files that may be copied and edited, instead of those predefined examples.

**Comment by StefanFW** [comment]
Good idea but I think these templates should not be part of the template itself. Maybe they could be downloadable from the wiki page.
